src/formatting_stack/strategies.clj

```diff
@@ -64,9 +80,12 @@
   The diff is compared against the `:target-branch` option."
   [& {:keys [target-branch impl files blacklist]
       :or   {target-branch "master"
-             impl          (impl/file-entries git-command "diff" "--name-only" target-branch)
+             impl          (impl/file-entries git-command "diff" "--name-status" target-branch)
              blacklist     (git-not-completely-staged :files [])}}]
   (->> impl
+       (remove (fn [s]
+                 (-> s (string/starts-with? "D\t"))))
+       (map #(string/replace-first % #"(A|C|D|M|R|T|U|X|B)\t" "")) ;; See: `git diff --help`, `diff-filter` section
```

Comment:
What about removing the first _word_ from the output? it seems less brittle

Reply:
> What about removing the first word from the output? i

Dunno, removing arbitrary stuff can also be brittle.

unit-testing the `replace-first` can be a middle ground